lib/nostrum/constants.ex

```diff
@@ -3,6 +3,7 @@ defmodule Nostrum.Constants do
 
   # REFERENCE: https://gist.github.com/SinisterRectus/9518f3e7d0d1ccb4335b2a0d389c30b0
   def base_url, do: Application.get_env(:nostrum, :api_url, "https://discordapp.com/api/v6")
+  def cdn_url, do: Application.get_env(:nostrum, :cdn_url, "https://cdn.discordapp.com")
```

Comment:
Can you change this function and the `base_url` one above it to just be a constant? Basically, just remove the `get_env` call since they're not needed anymore.

Reply:
Changed in 8e1b772